docs/_static/vcr/vcr_v2.yaml

```diff
@@ -191,21 +191,27 @@ paths:
       summary: "Revoke an issued credential"
       description: |
         Revoke a credential.
+        For a credential issued by did:nuts a revocation credential is published to the network
+        For a credential issued by did:web the revocation bit is set on the status list referenced in VC.credentialStatus
 
         error returns:
         * 400 - Credential can't be revoked. Most likely due to a missing private key.
+        * 404 - Credential not found
+        * 405 - Credential does not contain a StatusList2021Entry (only for did:web credentials)
```

Comment:
Not sure we need a separate status code for this, could just be 400 Bad Request. Same goes for the success response?

I think revocation is always "processed locally", since it just takes into effect when a verifier (be it VCs or PKIs) checks a credential or certificate. So you could just remove that sentence.